screeps-game-api/src/constants.rs

```diff
@@ -361,8 +361,8 @@ js_deserializable!(Color);
 ///
 /// *Note:* This constant's `TryFrom<Value>` and `Deserialize` implementations
 /// _only work with the integer constants_. If you're ever consuming strings
-/// such as `"plain"`, `"swamp"`, `"wall"`, you must convert those manually into
-/// `0`, `1` and `2` respectively.
+/// such as `"plain"`, `"swamp"`, `"wall"`, you can use the
```

Comment:
Only in JS though. Might be worthwhile to make a `TryFrom<String>` equivalent?

Reply:
Added! I gave it a `FromStr` implementation like some of the other constants.